src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -4113,17 +4062,27 @@ namespace VariableSpeedCoils {
                     OutTemp = FinalSysSizing(CurSysNum).OutTempAtCoolPeak;
                     rhoair = PsyRhoAirFnPbTdbW(OutBaroPress, MixTemp, MixHumRat, RoutineName);
                     MixEnth = PsyHFnTdbW(MixTemp, MixHumRat);
-                    MixWetBulb = PsyTwbFnTdbWPb(MixTemp, MixHumRat, OutBaroPress, RoutineName);
                     SupEnth = PsyHFnTdbW(SupTemp, SupHumRat);
 
+                    // design fan heat will be added to coil load
+                    Real64 FanCoolLoad = DataAirSystems::calcFanDesignHeatGain(DataFanEnumType, DataFanIndex, VolFlowRate);
+                    // inlet/outlet temp is adjusted after enthalpy is calculcated so fan heat is not double counted
+                    Real64 CpAir = PsyCpAirFnWTdb(MixHumRat, MixTemp);
```

Comment:
You've mentioned this before, I think, but why isn't MixEnth adjusted here?

Reply:
Given the test below at line 4082 for MixEnth > SupEnth it looks like MixEnth or SupEnth should have been updated here. However, the comment "The mixed air temp for zone equipment without an OA mixer is 0" implies it doesn't matter because MixEnth will always be greater than SupEnth (unless MixEnth=0 and then if IF at line 4082 would still work as intended).

Reply:
I wasn't asking about this particular coil type, but I think this is the general method elsewhere as well, no?  Leave the enthalpies, add the load, but adjust the entering or leaving conditions.  Just seems it would be cleaner to adjust the entering or leaving conditions first, then the enthalpies would already include the extra fan heat.